python/fusion_engine_client/messages/configuration.py

```diff
@@ -1160,6 +1160,44 @@ def calcsize(cls) -> int:
         return cls.SaveConfigMessageConstruct.sizeof()
 
 
+class SupportedIOInterfacesMessage(MessagePayload):
+    """!
+    @brief A list of transport interfaces supported by the device.
+    """
+    MESSAGE_TYPE = MessageType.SUPPORTED_IO_INTERFACES
+    MESSAGE_VERSION = 0
+
+    SupportedIOInterfacesMessageConstruct = Struct(
+        "num_interfaces" / Int8ul,
+        Padding(7),
+        "interfaces" / Array(this.num_interfaces, _InterfaceIDConstruct)
+    )
+
+    def __init__(self, action: SaveAction = SaveAction.SAVE):
+        self.interfaces : List[InterfaceID] = []
+
+    def pack(self, buffer: bytes = None, offset: int = 0, return_buffer: bool = True) -> (bytes, int):
+        packed_data = self.SupportedIOInterfacesMessageConstruct.build(self.__dict__)
```

Comment:
Does build populate `num_interfaces` automatically based on the interfaces array in the dict? I seem to recall seeing that done manually in pack functions for some other classes